### DISCUSSION TITLE
Repeat pattern

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#2517] [Q&A] [answered]
Here's a basic Excel feature that I would like to do with VisiData. I have this sheet:

![image](https://github.com/user-attachments/assets/8a674c5f-b090-43da-ba95-e63012874144)

I want to fill the weekdays column all the way to the end. I'm not expecting VisiData to automatically infer weekdays from dates, I just want to mark 7 days and extend it to the column as a repeating pattern, like you do in Excel. Any idea how to do that?

**Comment by saulpw** [comment, marked as answer]
Select 7 rows, use `gzy` to copy their cell values, then select the unfilled rows, and use `gzp` to paste them.  It'll automatically cycle the values.

**Reply by cool-RR** [reply]
Woohoo! Thank you.
